Documentation/compatibility/SerialPort-background-thread-exceptions.md

```diff
@@ -0,0 +1,41 @@
+## SerialPort background thread exceptions
+
+### Scope
+Minor
+
+### Version Introduced
+4.7.1
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+Background threads created with System.IO.Ports.SerialPort streams no longer terminate the process when OS exceptions are thrown. These background threads wait for OS events related to the active serial port and could crash in some cases, such as sudden removal of the serial port. 
+
+- [X] Quirked
+- [ ] Build-time break
+
+### Recommended Action
+
+For apps that target the .NET Framework 4.7.1, you can opt out of the exception handling if it is not desirable by adding the following to to the `<runtime>` section of your `app.config` file:
+
+   ```xml
+   <runtime>
+      <AppContextSwitchOverrides value="Switch.System.IO.Ports.DoNotCatchSerialStreamThreadExceptions=true" />
+   </runtime>
+   ```
+
+For apps that target earlier versions of the .NET Framework but run on the .NET Framework 4.7.1 or later, you can opt in to long path support by adding the following to to the `<runtime>` section of your `app.config` file:
```

Comment:
Is this supposed to be "you can opt in to the exception handling" instead of "you can opt in to long path support"?

Reply:
I think that is true -- @JeremyKuhne ?

Reply:
Yeah, whoops. :) Copy/paste fail.

Reply:
Fixed :).